tools/rna_tools/trna_mod/addCCA.xml

```diff
@@ -0,0 +1,22 @@
+<tool id="addCCA" name="add CCA tails to tRNA chromosomes" version="0.1.0">
```

Comment:
If you think this tool is easier, I would suggest to make the sequence which you add optional ... so that people can use this tool in more use-cases. You could even think of adding a sequence at the beginning as an option....